### DISCUSSION TITLE
Is it possible to force a user to change their password on first login by configuring a pam custom policy.

### DISCUSSION THREAD

**Discussion by muyu888** [libpwquality/libpwquality#76] [Q&A] [answered]
Is it possible to force a user to change their password on first login by configuring a pam custom policy.
Prerequisite:
The root, secadm and auditadm users have been created and initialized with passwords during system installation, while the rest of the ordinary users need to be created by the root user and have their passwords changed by the secadm user before they can log in.
The above prerequisite is the equalization of the separation of powers function, has been realized

**Comment by t8m** [comment, marked as answer]
This question is not really related to libpwquality or pam_pwquality. However the pam_unix module allows for such policy to be enforced with the help of shadow utils chage command. `chage -d 0 <user>` will make the user's password expired forcing it to be changed on next login.
